plugins/chat/spec/lib/guardian_extensions_spec.rb

```diff
@@ -299,8 +299,18 @@
       end
 
       context "when category has a channel" do
-        it "does not allow to delete the category" do
-          expect(staff_guardian).not_to be_able_to_delete_category(category)
+        context "when channel has no messages" do
+          it "allows to delete the category" do
```

Comment:
```suggestion
          it "allows a staff user to delete the category" do
```

When testing a guardian, I think it is important to state which user is carrying out an action. Perhaps we can put the assertion for a staff user into a `context`?